ipatests/prci_definitions/temp_commit.yaml

```diff
@@ -68,7 +68,7 @@ jobs:
       class: RunPytest
       args:
         build_url: '{fedora-latest/build_url}'
-        test_suite: test_integration/test_REPLACEME.py
+        test_suite: test_integration/test_ipahealthcheck.py::TestIpaHealthCheck
         template: *ci-master-latest
         timeout: 3600
```

Comment:
This test usually needs more than 1h to execute, see for instance https://pagure.io/freeipa/blob/master/f/ipatests/prci_definitions/nightly_latest.yaml#_1369: `timeout: 5400`

Reply:
@flo-renaud added the changes to yaml file, rerunning the job